ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/integration/error-handling/incorrect-url-tests.js

```diff
@@ -0,0 +1,40 @@
+describe("Error handling should present correctly to the user", () => {
+    after(function () {
+        cy.clearLocalStorage();
+    });
+
+	afterEach(() => {
+		cy.storeSessionData();
+	});
+
+	before(function () {
+		cy.login();
+	});
+
+    it("Should open first school in the list", () => {
+		cy.get("#school-name-0").click();
+		cy.get('*[href*="/confirm-school-trust-information-project-dates"]').should(
+			"be.visible"
+		);
+	});
+
+    // Raised under 80466
+    it.skip('Should display user-friendly error when incorrect return parameter passed [80466]',()=>{
+        let modifiedUrl
+        cy.get('[aria-describedby*=la-info-template-status]').click()
+        cy.url().then(url =>{
+            //Changes the current URL to:
+            //https://apply-to-become-an-academy-internal-dev.london.cloudapps.digital/task-list/<SOME_VALID_ID>/confirm-local-authority-information-template-dates?return=someInvalideParam/SomeInvalidPath
```

Comment:
```suggestion
            ///task-list/<SOME_VALID_ID>/confirm-local-authority-information-template-dates?return=someInvalideParam/SomeInvalidPath
```

removing mentions to specific urls